code/modules/food_and_drinks/food/snacks_burgers.dm

```diff
@@ -13,6 +13,19 @@
 	bonus_reagents = list("vitamin" = 1)
 	foodtype = GRAIN | MEAT
 
+/obj/item/reagent_containers/food/snacks/burger/plain/Initialize()
+	. = ..()
+	if(prob(1))
+		new/obj/effect/particle_effect/smoke(get_turf(src))
+		playsound(src, 'sound/effects/smoke.ogg', 50, TRUE)
+		visible_message("<span class='warning'>Oh, ye gods! [src] is ruined! But what if...?</span>")
+		name = "steamed ham"
```

Comment:
Steamed hams? Localized entirely within my research station?

Reply:
May I see it?

Reply:
No